Dockerfile.prod

```diff
@@ -1,21 +1,25 @@
-FROM node:14.21-alpine3.16 as build
+FROM docker.atl-paas.net/sox/micros-node-14:1.0.5 as build
+
+# root user for installs
+USER root
 
 # adding python for node-gyp
-RUN apk add g++ make python3
+RUN apt-get update && apt-get install -y g++ make python3
 
 # For coredumps
-RUN apk add gdb
-RUN apk add bash
+RUN apt-get install -y gdb
+
+# non-root user
+USER appuser
 
-# adding to solve vuln
-RUN apk add --update --upgrade busybox
-RUN apk add --update --upgrade libretls
-RUN apk add --update --upgrade openssl
-RUN apk add --update --upgrade zlib
+COPY . /opt/service
 
-COPY . /app
+WORKDIR /opt/service
 
-WORKDIR /app
+# Change ownership of the /opt/service directory to appuser
+USER root
```

Comment:
need to change ownership to allow these next steps to work as appuser...
using root the whole way through would likely work but its a big no no security wise. its slowwww so might see if we can do it a bit cleverer

Reply:
I don't get why the whole change to root/user back and forth. 
Does the `/opt/service` contains other stuff? If so, can we `COPY /opt/service/app` instead?

Reply:
This was exactly the comment i was looking for!! maybe? the chown call takes sooooo long.